Mongle/Tuist/ProjectDescriptionHelpers/Project+Templates.swift

```diff
@@ -19,7 +19,26 @@ public extension Project {
         var projectTargets: [Target] = []
         let schemes: [Scheme] = [
             Scheme.scheme(
-                name: "App",
+                name: "Mongle-prod",
+                shared: true,
+                buildAction: BuildAction.buildAction(targets: ["App"]),
```

Comment:
```suggestion
                buildAction: BuildAction.buildAction(targets: ["Mongle"]),
```
Mongle로 이름을 바꾸면 어떨까요?